clients/lsp-go.el

```diff
@@ -222,13 +237,37 @@ $GOPATH/pkg/mod along with the value of
   :group 'lsp-go
   :package-version '(lsp-mode "7.0.1"))
 
+(defcustom lsp-go-links-in-hover t
+  "If non-nil, hover documentation includes links."
+  :type 'boolean
+  :group 'lsp-go
+  :package-version '(lsp-mode "7.1"))
+
+(defcustom lsp-go-use-gofumpt nil
+  "If non-nil, use gofumpt formatting."
+  :type 'boolean
+  :group 'lsp-go
+  :package-version '(lsp-mode "7.1"))
+
+(defcustom lsp-go-goimports-local ""
+  "Equivalent of the goimports -local flag, which puts imports beginning with
+ this string after third-party packages.  It should be the prefix of the import
+ path whose imports should be grouped separately."
+  :type 'string
+  :group 'lsp-go
+  :package-version '(lsp-mode "7.1"))
+
 (lsp-register-custom-settings
  '(("gopls.usePlaceholders" lsp-go-use-placeholders t)
    ("gopls.hoverKind" lsp-go-hover-kind)
    ("gopls.buildFlags" lsp-go-build-flags)
    ("gopls.env" lsp-go-env)
    ("gopls.linkTarget" lsp-go-link-target)
-   ("gopls.codelenses" lsp-go-codelenses)))
+   ("gopls.codelenses" lsp-go-codelenses)
+   ("gopls.linksInHover" lsp-go-links-in-hover)
```

Comment:
booleans should have `t` as third argument.

Reply:
@yyoncho Fixed, thanks.